src/index.ts

```diff
@@ -0,0 +1,235 @@
+/**
+ * Type for (x, y) coordinate pair.
+ */
+ export type Point = { x: number, y: number };
+
+ /**
+  * Method signature for distance calculation.
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/icooper/curvereduce-js/pullRequest?prid=9960097)
```suggestion
 * Method signature for distance calculation.
```